tests/PerformanceHarness/__init__.py

```diff
@@ -0,0 +1,6 @@
+__all__ = ["log_reader", "performance_test_basic", "performance_test", "NodeosPluginArgs"]
+
+from .log_reader import blockData, trxData, chainData, scrapeTrxGenTrxSentDataLogs, JsonReportHandler, analyzeLogResults, TpsTestConfig, ArtifactPaths, LogAnalysis
```

Comment:
Not familiar with the leading dot in this format `from .module import Class`. Should be find otherwise this wouldn't run.

Reply:
@ericpassmore  - Here dot notation is being used to import names from a package module to the package namespace.